parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -1022,6 +1101,9 @@ public <D> CloseableIterable<D> build() {
         if (start != null) {
           optionsBuilder.withRange(start, start + length);
         }
+        if (fileDecryptionProperties != null) {
```

Comment:
Style: code blocks should be separated from the following statement by a newline.